ui/src/components/UserMenu/UserMenuButton.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { Component, Fragment } from 'react';
+import { withApollo } from 'react-apollo';
+import Avatar from '@material-ui/core/Avatar';
+import { withStyles } from '@material-ui/core/styles';
+import List from '@material-ui/core/List';
+import IconButton from '@material-ui/core/IconButton';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import Button from '@material-ui/core/Button';
+import AccountCircleIcon from 'mdi-react/AccountCircleIcon';
+import SignInDialog from '../SignInDialog';
+import { withAuth } from '../../utils/Auth';
+
+@withStyles(theme => ({
+  avatar: {
+    backgroundColor: theme.palette.secondary.main,
+  },
+  text: {
+    color: theme.palette.text.primary,
+    fontFamily: 'Roboto500',
+  },
+  icon: {
+    fill: theme.palette.common.white,
+    marginRight: theme.spacing.unit,
+    marginLeft: theme.spacing.unit,
```

Comment:
Remove both `marginLeft` and `marginRight`. Add a marginLeft on the `UserMenuButton` instead. See other comment.